src/components/SettingDialog.vue

```diff
@@ -49,25 +49,101 @@
-                  toggle-color="blue"
-                  :options="[
-                    { label: 'UTF-8', value: 'UTF-8' },
-                    { label: 'Shift_JIS', value: 'Shift_JIS' },
-                  ]"
-                />
-              </div>
+              <q-tab-panels v-model="tab">
+                <q-tab-panel name="encoding">
+                  <div class="q-pa-md">
+                    <q-btn-toggle
+                      v-model="fileEncoding"
+                      toggle-color="blue"
+                      :options="[
+                        { label: 'UTF-8', value: 'UTF-8' },
+                        { label: 'Shift_JIS', value: 'Shift_JIS' },
+                      ]"
+                    />
+                  </div>
+                </q-tab-panel>
+                <q-tab-panel name="directory">
+                  <q-toggle
+                    name="enabled"
+                    align="left"
+                    dense
+                    color="blue"
+                    :model-value="simpleMode.enabled"
+                    :label="simpleMode.enabled ? '有効' : '無効'"
+                    @update:model-value="
+                      handleSimpleModeChange('enabled', $event)
+                    "
+                  />
+                  <q-input
+                    unelevated
+                    dense
+                    bottom-slots
+                    v-model="simpleMode.dir"
+                    label="デフォルトのフォルダ"
+                    @update:model-value="handleSimpleModeChange('dir', $event)"
+                    color="blue"
+                  >
+                    <template v-slot:append>
+                      <q-btn
+                        square
+                        dense
+                        flat
+                        color="blue"
+                        icon="folder_open"
+                        @click="onOpeningFileExplore"
+                      >
+                        <q-tooltip
+                          :delay="500"
+                          class="bg-blue text-body2"
+                          anchor="bottom right"
+                        >
+                          フォルダー選択
+                        </q-tooltip>
+                      </q-btn>
+                    </template>
+                  </q-input>
+                  設定したフォルダーに書き出す
+                </q-tab-panel>
+
+                <q-tab-panel name="avoid">
+                  <div class="q-py-sm">
+                    <q-checkbox
+                      class="q-pl-lg"
+                      label="上書き防止"
+                      dense
+                      color="blue"
+                      :model-value="simpleMode.avoid"
+                      @update:model-value="
+                        handleSimpleModeChange('avoid', $event)
+                      "
+                    />
+                  </div>
+                  <q-separator />
+                  <div class="q-pt-sm">
+                    上書きの代わりにファイル名に番号をつける
```

Comment:
「上書きせずにファイルを連番にします。」というのはどうでしょう？
How about "Sets sequential numbering of files without overwriting."?

Reply:
トグルボタンの下に説明が書かれていますが、人の目は上から下に移動するので、説明を上に移動すると良いかもしれません。
The description is written below the toggle button, but since people's eyes move from top to bottom, it might be a good idea to move the description to the top.